pkg/osquery/runtime/osquery_instance_history/history.go

```diff
@@ -0,0 +1,65 @@
+package osquery_instance_history
+
+import (
+	"errors"
+	"time"
+)
+
+const max_instances = 10
```

Comment:
prefer camel case

Reply:
switching to camel case ... the snake case I think is left over from C# / java in my brain where constants were usually named `LIKE_THIS`